gtk/src/subclass/application.rs

```diff
@@ -49,8 +48,8 @@ impl<T: GtkApplicationImpl> GtkApplicationImplExt for T {
     }
 }
 
-unsafe impl<T: GtkApplicationImpl> IsSubclassable<T> for ApplicationClass {
-    fn override_vfuncs(&mut self) {
+unsafe impl<T: GtkApplicationImpl> IsSubclassable<T> for Application {
+    fn override_vfuncs(class: &mut ::glib::object::Class<Self>) {
```

Comment:
I hope we get custom self types at some point, `self: &mut Class<Self>` (compare `Future::poll`) would look nicer :)

Reply:
The unstable arbitrary self types feature in Rust, as implemented, only works when the type implements `Deref` to `self`. So assuming that's how a stable feature would work, I don't think it would help here. Unfortunately.

Reply:
Ah too bad. We can live with that, I guess